docs/src/man/missing.md

```diff
@@ -1,177 +1,177 @@
-
-```jldoctest missings
-julia> df = DataFrame(i = 1:5,
-                      x = [missing, 4, missing, 2, 1],
-                      y = [missing, missing, "c", "d", "e"])
-5×3 DataFrame
- Row │ i      x        y
-     │ Int64  Int64?   String?
-─────┼─────────────────────────
-   1 │     1  missing  missing
-   2 │     2        4  missing
-   3 │     3  missing  c
-   4 │     4        2  d
-   5 │     5        1  e
-
-julia> dropmissing(df)
-2×3 DataFrame
- Row │ i      x      y
-     │ Int64  Int64  String
-─────┼──────────────────────
-   1 │     4      2  d
-   2 │     5      1  e
-```
-
-One can specify the column(s) in which to search for rows containing `missing` values to be removed.
-
-```jldoctest missings
-julia> dropmissing(df, :x)
-3×3 DataFrame
- Row │ i      x      y
-     │ Int64  Int64  String?
-─────┼───────────────────────
-   1 │     2      4  missing
-   2 │     4      2  d
-   3 │     5      1  e
-```
-
-By default the `dropmissing` and `dropmissing!` functions keep the
-`Union{T,Missing}` element type in columns selected for row removal. To remove
-the `Missing` part, if present, set the `disallowmissing` option to `true` (it
-will become the default behavior in the future).
-
-```jldoctest missings
-julia> dropmissing(df, disallowmissing=true)
-2×3 DataFrame
- Row │ i      x      y
-     │ Int64  Int64  String
-─────┼──────────────────────
-   1 │     4      2  d
-   2 │     5      1  e
-```
-
-The [Missings.jl](https://github.com/JuliaData/Missings.jl) package provides a
-few convenience functions to work with missing values.
-
-The function `Missings.replace` returns an iterator which replaces `missing`
-elements with another value:
-
-```jldoctest missings
-julia> using Missings
-
-julia> Missings.replace(x, 1)
-Missings.EachReplaceMissing{Array{Union{Missing, Int64},1},Int64}(Union{Missing, Int64}[1, 2, missing], 1)
-
-julia> collect(Missings.replace(x, 1))
-3-element Array{Int64,1}:
- 1
- 2
- 1
-
-julia> collect(Missings.replace(x, 1)) == coalesce.(x, 1)
-true
-
-```
-
-The function `nonmissingtype` returns the element-type `T` in `Union{T, Missing}`.
-
-```jldoctest missings
-julia> eltype(x)
-Union{Int64, Missing}
-
-julia> nonmissingtype(eltype(x))
-Int64
-
-```
-
-The `missings` function constructs `Vector`s and `Array`s supporting missing
-values, using the optional first argument to specify the element-type.
-
-```jldoctest missings
-julia> missings(1)
-1-element Array{Missing,1}:
- missing
-
-julia> missings(3)
-3-element Array{Missing,1}:
- missing
- missing
- missing
-
-julia> missings(1, 3)
-1×3 Array{Missing,2}:
- missing  missing  missing
-
-julia> missings(Int, 1, 3)
-1×3 Array{Union{Missing, Int64},2}:
- missing  missing  missing
-
-```
-
-See the [Julia manual](https://docs.julialang.org/en/v1/manual/missing/) for more information about missing values.
+# Missing Data
```

Comment:
why is it all changed here?

Reply:
especially that it seems that this file is almost unchanged.

Reply:
Yeah, this is weird. When I look in VS code's diff view, the only thing changed is line 105, but when I use `git diff`, it shows the same thing as this.

I think this is a git bug. If you checkout master, add that one comma and then do `git diff`, it says the whole file is changed.

Reply:
OK - just wanted to make sure as this is hard to review.